src/sentry/api/endpoints/team_release_count.py

```diff
@@ -0,0 +1,37 @@
+from collections import defaultdict
+
+from django.db.models import Count
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.models import Project, Release
+
+
+class TeamReleaseCountEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Returns a dict of team projects, and a time-series list of release counts for each.
+        """
+        project_list = Project.objects.get_for_team_ids(team_ids=[team.id])
+        start, end = get_date_range_from_params(request.GET)
+
+        bucketed_releases = (
+            Release.objects.filter(
+                projects__in=project_list, date_added__gte=start, date_added__lte=end
+            )
+            .distinct()
+            .annotate(bucket=TruncDay("date_added"))
+            .values("projects", "bucket")
```

Comment:
What does `values("projects")` return here? Just a `project_id` per row?

Reply:
Yeah, exactly that

Reply:
I just realised why this stuck out to me - this will cause us to have a row per project/release. I guess the problem here is you have two separate goals:

You want unique count of releases per day.
You want count of releases per project.

Based on the designs you don't need to return results bucketed by project/day though. Don't you want two separate values to be returned?
 - Releases created per day for this team
 - Releases created per project for this team